metals/src/main/scala/scala/meta/internal/metals/formatting/IndentOnPaste.scala

```diff
@@ -146,6 +147,7 @@ case class IndentOnPaste(userConfig: () => UserConfiguration)
       case (line, _) =>
         PastedLine.plainOrEmpty(normalizeSpacesAndTabs(line, opts))
     }
+    @nowarn("msg=The outer reference")
```

Comment:
So aren't we silencing this same warning in other places in the `build.sbt`, but specifically here as well? I find it a bit confusing if we silence it in some packages in the `build.sbt` and then in this file here. Could we had this place in there as well?

Reply:
Good catch!
I started with `@nowarns` in files but it yielded a compilation error in mtags since `nowarn` was introduced in some specific version of scala and this one slipped through. 
I'll move it to the `build.sbt` as others.